target/portfolio-1/js/map.js

```diff
@@ -0,0 +1,25 @@
+function initMap() {
+  var myLatlng = {lat: -0.21874964312341205, lng: 12.221368164062483};
```

Comment:
map.js is not a file anymore. I moved this to createGame.js and this is the same function as initMapCreateGame() just I added it to another file since different pages have different maps.

It is not a test value. This is for when the map loads very zoomed out, we want the origin of the map to be the center and that is why I have it (changed since this) to be (0, 0). This is to show that we are very biased toward any country.